### PR TITLE
Hide some "private" functions

### PR DESCRIPTION
**Pull request by vojtechtrefny**
These shouldn't be publicly visible.

### CODE REVIEW

File: src/plugins/fs/ext.c [hunk 1]
```diff
@@ -148,7 +148,8 @@ static gboolean extract_e2fsck_progress (const gchar *line, guint8 *completion)
  * Returns: whether the @tech-@mode combination is available -- supported by the
  *          plugin implementation and having all the runtime dependencies available
  */
-gboolean bd_fs_ext_is_tech_avail (BDFSTech tech UNUSED, guint64 mode, GError **error) {
+gboolean __attribute__ ((visibility ("hidden")))
```
Comment (tbzatek) [inline src/plugins/fs/ext.c]: You may also use `G_GNUC_INTERNAL` for that. According to docs, you'll get support to _Sun Studio compiler_ ;-)

Also the glib docs recommend to move the macro at the very beginning of the declaration.

File: scripts/boilerplate_generator.py [hunk 2]
```diff
@@ -196,12 +196,12 @@ def get_funcs_info(fn_infos, module_name):
...
-    ret += ("const gchar* const* get_{0}_functions (void) {{\n".format(module_name) +
+    ret += ("static const gchar* const* get_{0}_functions (void) {{\n".format(module_name) +
             "    return {0}_functions;\n".format(module_name) +
             "}\n\n")
 
     ret += "static const guint8 {0}_num_functions = {1};\n\n".format(module_name, len(fn_infos))
-    ret += ("guint8 get_{0}_num_functions (void) {{\n".format(module_name) +
+    ret += ("static guint8 get_{0}_num_functions (void) {{\n".format(module_name) +
             "    return {0}_num_functions;\n".format(module_name) +
             "}\n\n")
```
Comment (vpodzime) [inline scripts/boilerplate_generator.py]: These two should probably be removed, we are not using them anywhere, AFAICT.

Comment (vojtechtrefny) [reply scripts/boilerplate_generator.py]: Ok, removed.

File: src/plugins/fs/ext.c (hunk 1 above)
Comment (vojtechtrefny) [reply src/plugins/fs/ext.c]: > According to docs, you'll get support for *Sun Studio compiler* ;-)

Wow, we definitely need that :-)

> Also the glib docs recommend to move the macro at the very beginning of the declaration.

I'd like to keep this consistent with other "hidden" functions where we use the macro like this.

Comment (tbzatek) [reply src/plugins/fs/ext.c]: Fair enough :-)

**Pull request merged**